.env.example

```diff
@@ -0,0 +1,149 @@
+# =============================================================================
+# ROFL Price Oracle - Off-Chain Aggregation Configuration
+# =============================================================================
+#
+# This oracle aggregates prices from multiple off-chain sources using median
+# calculation with outlier detection, then submits to on-chain aggregator contracts.
+#
+# Architecture:
+#   Multiple APIs → Median Aggregation → On-Chain Submission
+#   (Coinbase, Kraken, CoinGecko, etc.) → median → Aggregator Contract
+#
+# =============================================================================
+
+# -----------------------------------------------------------------------------
+# Trading Pairs
+# -----------------------------------------------------------------------------
+# Comma-separated list of trading pairs to track.
+# Format: base/quote (e.g., btc/usd, eth/usd, rose/usd)
+PAIRS=usdt/usd,rose/usd
```

Comment:
This is tricky since CEXes have different trading pairs: Some have rose/usdc, others have rose/usdt, maybe even rose/usd. How do you handle those? Should we treat USDC/USDT/USD as the same currency?